lib/tests/streamlit/streamlit_test.py

```diff
@@ -397,45 +397,23 @@ def test_st_exception(self, show_error_details: bool):
             # streamlit.elements.exception_element
             self.assertEqual(el.exception.stack_trace, [])
 
-    def test_st_header(self):
-        """Test st.header."""
-        st.header("some header")
-
-        el = self.get_delta_from_queue().new_element
-        self.assertEqual(el.heading.body, "some header")
-        self.assertEqual(el.heading.tag, "h2")
-
-    def test_st_header_with_anchor(self):
-        """Test st.header with anchor."""
-        st.header("some header", anchor="some-anchor")
-
-        el = self.get_delta_from_queue().new_element
-        self.assertEqual(el.heading.body, "some header")
-        self.assertEqual(el.heading.tag, "h2")
-        self.assertEqual(el.heading.anchor, "some-anchor")
-
     def test_st_help(self):
         """Test st.help."""
-        st.help(st.header)
+        st.help(st.help)
```

Comment:
How is this change related to anchor?

Reply:
This test checks the signature of the `st.header` method, which has changed, so I updated the test.
<img width="1310" alt="Screenshot 2023-02-28 at 19 15 17" src="https://user-images.githubusercontent.com/78743291/221943448-4831db78-0481-4a3c-9998-c2ed30700c7b.png">